pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "todo"
+version = "0.1.0"
+description = ""
+authors = ["Colin Budd <colin.budd@capita.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+astroid = "^2.4.2"
+certifi = "^2020.6.20"
+chardet = "^3.0.4"
+click = "^7.1.2"
+colorama = "^0.4.3"
+Flask = "^1.1.1"
+idna = "^2.10"
+isort = "^4.3.21"
+itsdangerous = "^1.1.0"
+Jinja2 = "^2.11.2"
+lazy-object-proxy = "^1.4.3"
+MarkupSafe = "^1.1.1"
+mccabe = "^0.6.1"
+python-dotenv = "^0.12.0"
+requests = "^2.24.0"
+six = "^1.15.0"
+toml = "^0.10.1"
+urllib3 = "^1.25.10"
+Werkzeug = "^1.0.1"
+wrapt = "^1.12.1"
```

Comment:
You don't need to include nested dependencies (the lock file will still prevent any surprise changes there). I think this can be reduced to python, Flask, python-dotenv, requests